Source/Krypton Components/Krypton.Toolkit/General/GlobalStaticValues.cs

```diff
@@ -30,6 +30,8 @@ public class GlobalStaticValues
         /// <summary>The global button padding</summary>
         public const int GLOBAL_BUTTON_PADDING = 10;
 
+        internal const int DEFAULT_PADDING = 10;
```

Comment:
Does this need to be scaled for different screen settings ?

Reply:
Yes, I've left a todo note in the affected `if` statement